utils/db.ts

```diff
@@ -132,24 +137,24 @@ export async function setUserSubscription(
   isSubscribed: boolean,
 ) {
   const key = ["users", id];
-  const userRes = await kv.get<UserValue>(key);
+  const userRes = await kv.get<User>(key);
 
   if (userRes.value === null) throw new Error("User with ID does not exist");
 
   const res = await kv.atomic()
     .check(userRes)
-    .set(key, { ...userRes.value, isSubscribed } as UserValue)
+    .set(key, { ...userRes.value, isSubscribed } as User)
```

Comment:
This seems only updating the user at `users, id` key and the other user data at `users_by_stripe_customer` is not update. I don't think saving the same entity (user) in 2 different places are good idea (That will be bug prone, and the broken data will be hard to fix.)